yogstation/code/modules/mob/living/simple_animal/borer/borer_powers.dm

```diff
@@ -0,0 +1,463 @@
+	set desc = "Slither out of your host."
+
+	if(!victim)
+		to_chat(src, "<span class='userdanger'>You are not inside a host body.</span>")
+		return
+
+	if(stat != CONSCIOUS)
+		to_chat(src, "<span class='userdanger'>You cannot leave your host in your current state.</span>")
+
+	if(!victim || !src) return
+
+	if(leaving)
+		leaving = 0
+		to_chat(src, "<span class='userdanger'>You decide against leaving your host.</span>")
+		return
+
+	to_chat(src, "<span class='userdanger'>You begin disconnecting from [victim]'s synapses and prodding at their internal ear canal.</span>")
+
+	if(victim.stat != DEAD)
+		to_chat(victim, "<span class='userdanger'>An odd, uncomfortable pressure begins to build inside your skull, behind your ear...</span>")
+
+	leaving = 1
+
+	spawn(100)
+
+		if(!victim || !src) return
+		if(!leaving) return
+		if(controlling) return
+
+		if(src.stat != CONSCIOUS)
+			to_chat(src, "<span class='userdanger'>You cannot release your host in your current state.</span>")
+			return
+
+		to_chat(src, "<span class='userdanger'>You wiggle out of [victim]'s ear and plop to the ground.</span>")
+		if(victim.mind)
+			to_chat(victim, "<span class='danger'>Something slimy wiggles out of your ear and plops to the ground!</span>")
+			to_chat(victim, "<span class='danger'>As though waking from a dream, you shake off the insidious mind control of the brain worm. Your thoughts are your own again.</span>")
+		leave_victim()
+
+
+
+/mob/living/simple_animal/borer/verb/jumpstart()
+	set category = "Borer"
+	set name = "Jumpstart Host"
+	set desc = "Brings your host back from the dead."
+
+	if(!victim)
+		to_chat(src, "<span class='warning'>You need a host to be able to use this.</span>")
+		return
+
+	if(docile)
+		to_chat(src, "<span class='warning'>You are feeling too docile to use this!</span>")
+		return
+
+	if(chemicals < 250)
+		to_chat(src, "<span class='warning'>You need 250 chemicals to use this!</span>")
+		return
+
+	if(!checkStrength())
+		return 0
+
+	if(victim.stat == DEAD)
+		GLOB.dead_mob_list -= victim
+		GLOB.alive_mob_list += victim
+		victim.tod = null
+		victim.setToxLoss(0)
+		victim.setOxyLoss(0)
+		victim.setCloneLoss(0)
+		victim.SetKnockdown(0)
+		victim.SetStun(0)
+		victim.SetParalyzed(0)
+		victim.radiation = 0
+		victim.heal_overall_damage(victim.getBruteLoss(), victim.getFireLoss())
+		victim.reagents.clear_reagents()
+		if(istype(victim,/mob/living/carbon/human))
+			var/mob/living/carbon/human/H = victim
+			H.restore_blood()
+			H.remove_all_embedded_objects()
+		//victim.update_canmove()
+		victim.med_hud_set_status()
+		victim.med_hud_set_health()
+		victim.stat = CONSCIOUS
+		log_game("[src]/([src.ckey]) has revived [victim]/([victim.ckey]")
+		chemicals -= 250
+		to_chat(src, "<span class='notice'>You send a jolt of energy to your host, reviving them!</span>")
+		victim.grab_ghost(force = TRUE) //brings the host back, no eggscape
+		victim <<"<span class='notice'>You bolt upright, gasping for breath!</span>"
```

Comment:
is there a point to not just using the revive() proc here?

Reply:
revive() will restore limbs I think